_examples/fruits_test.go

```diff
@@ -37,9 +37,15 @@ func TestFruits(t *testing.T) {
 		Expect().
 		Status(http.StatusNoContent).NoContent()
 
-	e.GET("/fruits").
+	fruitsResult := e.GET("/fruits").
 		Expect().
-		Status(http.StatusOK).JSON().Array().ContainsOnly("orange", "apple")
+		Status(http.StatusOK).JSON().Array()
+
+	fruitsResult.ContainsOnly("orange", "apple")
+	fruitsResult.ContainsAny("orange", "melon")
+	fruitsResult.Every(func(index int, value *httpexpect.Value) {
+		value.String().NotEmpty()
+	})
```

Comment:
I'd reorder this: first call Every (a more generic check - all elements are non-empty strings), then call ContainsXxx (more specific checks).

Reply:
 👍🏽